### PR TITLE
Clean out unnecessary stuff and format code

### PR DESCRIPTION
**Pull request by bhollis**
Removes firebase dependency and push notification stuff, and reformats the code.

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -1,11 +1,7 @@
 # Uncomment the next line to define a global platform for your project
-platform :ios, '13.0'
+platform :ios, '14.0'
```
Comment (kyleshay) [inline Podfile:2]: Does this break support for 13 / was there a reason for bumping?

File: DIM/Settings.swift [hunk 2]
```diff
@@ -5,10 +5,8 @@ struct Cookie {
     var value: String
 }
 
-let gcmMessageIDKey = "00000000000" // update this with actual ID if using Firebase 
-
 // URL for first launch
-let rootUrl = URL(string: "https://app.destinyitemmanager.com/index.html?utm_source=homescreen")!
+let rootUrl = URL(string: "https://app.destinyitemmanager.com/index.html?utm_source=ios_app")!
```
Comment (kyleshay) [inline DIM/Settings.swift:9]: 👍

**Review by kyleshay — APPROVED** [review]
thanks! I should pull a fresh project from the pwa builder and diff at some point to see if anything is worth pulling in.

File: Podfile (hunk 1 above)
Comment (kyleshay) [reply Podfile:2]: https://github.com/DestinyItemManager/DIM/pull/8933  partially related?

Comment (bhollis) [reply Podfile:2]: We increased the minimum iOS version for DIM to 14.0 a while back.

**Pull request merged**